src/user/user.router.spec.ts

```diff
@@ -78,7 +78,7 @@ describe('User Router', () => {
       tempUser = testUsers[0];
       chai.request(server)
         .delete('/api/user/' + testUsers[0]._id)
-        .end((err: Error, res: Response) => {
+        .end((err: Error, res) => {
```

Comment:
?

Reply:
It was my bad with the Microsoft linter..

Reply:
But why does that change in tal's branch?

Reply:
Because at the time a similar problem caused a compilation error. You are right that it should not be changed- but I believe I am the one who caused the change.